code/datums/diseases/transformation.dm

```diff
@@ -67,10 +67,10 @@
 		affected_mob.create_log(MISC_LOG, "has transformed into [new_mob], due to having the virus \"[src]\"")
 		if(istype(new_mob))
 			new_mob.a_intent = "harm"
-			if(affected_mob.mind)
-				affected_mob.mind.transfer_to(new_mob)
 			if(isrobot(new_mob))
 				new_mob.rename_self("Cyborg", TRUE, TRUE)
```

Comment:
IMO, this should be changed to be after, otherwise I don't think theyll get the prompt, right?

Reply:
Good point, changed!

Reply:
Tested both xenomicrobes and nanomachines, both worked as intended